src/openshift/cluster.ts

```diff
@@ -72,14 +72,20 @@ export class Cluster extends OpenShiftItem {
 
     static async login(): Promise<string> {
         const response = await Cluster.requestLoginConfirmation();
+        const loginActions = [
+            {
+                label: 'Credentials',
+                description: 'Log in to the given server using credentials'
+            },
+            {
+                label: 'Token',
+                description: 'Login using bearer token for authentication to the API server'
```

Comment:
What if we change it to `Log in to the given server using bearer token` for consistency?

Reply:
Fixed.